Containers/talk-recording/start.sh

```diff
@@ -13,7 +13,7 @@ elif [ -z "$INTERNAL_SECRET" ]; then
 fi
 
 if [ -z "$HPB_DOMAIN" ]; then
-    export HPB_DOMAIN="$NC_DOMAIN"
+    export HPB_DOMAIN
```

Comment:
I don't understand the sense of this, you want to say, if $HPB_DOMAIN is not set, then export it, without setting a value for it?

Reply:
the current logic is: if $HPB_DOMAIN is unset, it will be set to the same value as NC_DOMAIN

Reply:
--- closed.